src/apps/reservation-list/modal/reservation-details/physical-list-details.tsx

```diff
@@ -95,15 +92,15 @@ const PhysicalListDetails: FC<PhysicalListDetailsProps & MaterialProps> = ({
 
   const changeExpiryDate = useCallback(
     (e: ChangeEvent<HTMLSelectElement>) => {
-      const newEpiryDate = formatInterestPeriods.filter(
+      const newEpiryDate = interstPeriods.filter(
```

Comment:
Was const `newEpiryDate` meant to be `newExpiryDate` ?

Reply:
ha, 
<img width="782" alt="image" src="https://github.com/danskernesdigitalebibliotek/dpl-react/assets/15377965/25be6cbd-1593-42a2-bf17-6d603cf5e304">

I'll rename